src/cocotb_tools/runner.py

```diff
@@ -159,10 +173,33 @@ def build(
     ) -> None:
         """Build the HDL sources.
 
+        With mixed language simulators, *sources* will be built,
+        followed by *vhdl_sources*, then *verilog_sources*.
+        With simulators that only support either VHDL or Verilog, *sources* will be built,
+        followed by *vhdl_sources* and *verilog_sources*, respectively.
+
+        If your sources files use an atypical file extension,
+        use :class:`VHDL` and :class:`Verilog` to *tag* the path as a VHDL or Verilog source file, respectively.
```

Comment:
Other italicized words are arguments to build. If you want emphasis you could bold it.
```suggestion
        use :class:`VHDL` and :class:`Verilog` to tag the path as a VHDL or Verilog source file, respectively.
```